lab_3_keywords_textrank/main.py

```diff
@@ -564,7 +676,8 @@ def get_top_keywords(self, n_keywords: int) -> tuple[int, ...]:
             tuple[int, ...]
                 top n most important tokens in the encoded text
         """
-        pass
+        sorted_by_key = sorted(self._scores)
```

Comment:
although in later versions of python dictionaries are indeed ordered, traditionally they are thought of as unordered 
so for the mark 10 I would prefer that you find another way to establish double sorting order

hint: look into `key` `argument in `sorted` function, it _is_ possible to put there the primary and secondary sorting orders